lib/dialects/mssql/query.js

```diff
@@ -164,8 +164,8 @@ module.exports = (function() {
   Query.prototype.handleShowTablesQuery = function(results) {
     return results.map(function(resultSet) {
       return {
-        tableName: resultSet.TABLE_NAME,
-        schema: resultSet.TABLE_SCHEMA
+        tableName: resultSet.tableName,
+        schema: resultSet.schema
```

Comment:
https://github.com/sequelize/sequelize/blob/master/lib/query-interface.js#L321-L335
here call  tableName: resultSet.tableName

but mssql dialect return  resultSet.TABLE_NAME and
when i call QueryInterface.prototype.describeTable i have undefined input parameters  https://github.com/sequelize/sequelize/blob/master/lib/query-interface.js#L337 
of function.